src/odemis/driver/andorcam2.py

```diff
@@ -2938,6 +3058,9 @@ def IsTriggerModeAvailable(self, mode):
         if _val(mode) not in self._supported_triggers:
             raise AndorV2Error(20078, "DRV_INVALID_MODE")
 
+    def SetTriggerInvert(self, inverted):
+        return
```

Comment:
This does nothing, is a pass not better instead of a return? Why does it do nothing? Maybe that can be commented

Reply:
It does nothing because we don't need to simulate the inversion of the hardware trigger, as it's all simulated. I've added a comment.